### PR TITLE
[No Ticket][Risk=no] UI cleanup

### PR DESCRIPTION
**Pull request by blrubenstein**
This PR changes more of our colors to use the color palette colors (Causing some style changes on the pages) as well as removes unused code.

### CODE REVIEW

**Comment by calbach** [comment]
Could you add a PR description? -- Is this a code style cleanup (and a functional no-op), or is this meant to make UI styling changes?

**Comment by blrubenstein** [comment]
@calbach done.

File: ui/src/app/components/inputs.tsx [hunk 1]
```diff
@@ -103,10 +102,9 @@ export const TextArea = ({style = {}, onChange, invalid = false, ...props}) => {
     onChange={onChange ? (e => onChange(e.target.value)) : undefined}
     style={{
       width: '100%',
-      borderColor: '#c5c5c5', borderWidth: 1, borderStyle: 'solid', borderRadius: 3,
+      borderColor: inputBorderColor, borderWidth: 1, borderStyle: 'solid', borderRadius: 3,
       padding: '0.25rem 0.5rem',
-      backgroundColor: '#fff',
-      ...(invalid ? styles.unsuccessfulInput : {}),
```
Comment (calbach) [inline ui/src/app/components/inputs.tsx]: invalid appears to no longer be used - please remove from props here and above

File: ui/src/app/components/tables.tsx [hunk 2]
```diff
@@ -1,125 +1,13 @@
...
-    display: 'flex', alignItems: 'center', height: rowHeight,
-    border: '1px solid #ccc', backgroundColor: '#fafafa', fontSize: 13
-  },
   paddedTable: {
     color: colors.primary, fontSize: 13, lineHeight: '24px',
     background: colors.white, padding: '0.2rem 0.5rem', display: 'flex'
   }
 };
 
-export const SimpleTable = ({columns, rowCount, rowProps}) => {
```
Comment (calbach) [inline ui/src/app/components/tables.tsx]: How did you identify this dead code? IDE? Script?

Comment (blrubenstein) [reply ui/src/app/components/tables.tsx]: IDE, and then find all in directory on UI showed no uses.

**Pull request merged**